app/views/declaration/representative_details.scala.html

```diff
@@ -92,8 +92,8 @@
             field = form("details.eori"),
             labelKey = "supplementary.representative.eori.legend",
             hintKey = Some("supplementary.representative.eori.hint"),
-            isPageHeading = true,
-            headingClasses = "govuk-label--m"
+            isPageHeading = false,
+            headingClasses = ""
```

Comment:
I meant you can get rid of the whole "headingClass" line.   The value isn't used if isPageHeading is false.